examples/src/main/java/com/mapbox/navigation/examples/ui/NavigationViewActivity.kt

```diff
@@ -101,7 +101,7 @@ class NavigationViewActivity : AppCompatActivity(), OnNavigationReadyCallback,
                 optionsBuilder.directionsRoute(route)
                 optionsBuilder.shouldSimulateRoute(true)
                 optionsBuilder.bannerInstructionsListener(this)
-                optionsBuilder.muteVoiceGuidance(true)
```

Comment:
Should we keep it to showcase how to use the `muteVoiceGuidance` API?

Reply:
yeah, I was thinking about this. But I am afraid the developer will be confusing about why they can't hear the voice instruction.